Cannot find module 'babel-runtime/core-js/json/stringify'

When running the following command:

```bash
git-stats --raw | git-stats-html -o out.html
```

I have the following error:

```
module.js:487
    throw err;
    ^

Error: Cannot find module 'babel-runtime/core-js/json/stringify'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/git-stats-html/node_modules/ajs/lib/index.js:3:18)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
events.js:182
      throw er; // Unhandled 'error' event
      ^

Error: write EPIPE
    at exports._errnoException (util.js:1016:11)
    at WriteWrap.afterWrite [as oncomplete] (net.js:850:14)
```

_____

`git-stats` works perfectly by the way.

I'm using node.js 8.1.2.

--- a/.gitignore
+++ b/.gitignore
@@ -3,3 +3,4 @@
 *~
 *.log
 node_modules
+*.env

--- a/DOCUMENTATION.md
+++ b/DOCUMENTATION.md
@@ -6,6 +6,7 @@ You can see below the API reference of this module.
 Turn git-stats result into HTML output.
 
 #### Params
+
 - **Object** `json`: The JSON returned by `git-stats --raw`
 - **Function** `cb`: The callback function.
 

--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
     "output"
   ],
   "license": "MIT",
-  "version": "1.0.3",
+  "version": "1.0.4",
   "main": "lib/index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"

